packages/web3-providers/src/common.ts

```diff
@@ -0,0 +1,8 @@
+export enum Days {
```

Comment:
I don't think this type is related to any providers.  However, we should treat it to types of plugins.

`/packages/mask/src/plugins/Trader/types/trending.ts`

Reply:
`NFTScan` provider and `cmc` provider rely on it.

Reply:
Okay then, what about placing it `packages/web3-providers/src/types/Trending.ts`. Much better then a dangling `export`. It is harmful.

Reply:
Updated.